caas/broker.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package caas
+
+import "github.com/juju/juju/environs"
+
+// NewBrokerFunc returns a Broker.
+type NewBrokerFunc func(environs.CloudSpec) (Broker, error)
+
+// NewOperatorConfigFunc functions return the agent config to use for
+// a CAAS jujud operator.
+type NewOperatorConfigFunc func(appName string) ([]byte, error)
```

Comment:
can we have it return an OperatorConfig struct? surely it's going to grow to include more than just the agent.conf contents?

Reply:
done